src/rdkafka_cgrp.c

```diff
@@ -2742,26 +2747,30 @@ static void rd_kafka_cgrp_group_leader_reset (rd_kafka_cgrp_t *rkcg,
 
 
 /**
- * @brief Group is rebalancing, trigger rebalance callback to application,
+ * @brief Initiate a rebalance. Trigger rebalance callback to application,
```

Comment:
Technically it is the group coordinator that initiates a rebalance, not the client, so I think the former wording is more correct.